recipes/cpython/all/conanfile.py

```diff
@@ -0,0 +1,491 @@
+        self.run("{} {}".format(python_built, python_args), run_environment=True)
+
+        for file in os.listdir(install_prefix):
+            if re.match("vcruntime.*", file):
+                os.unlink(os.path.join(install_prefix, file))
+                continue
+        os.unlink(os.path.join(install_prefix, "LICENSE.txt"))
+        for file in os.listdir(os.path.join(install_prefix, "libs")):
+            if not re.match("python.*", file):
+                os.unlink(os.path.join(install_prefix, "libs", file))
+
+    def _msvc_package_copy(self):
+        build_path = self._msvc_artifacts_path
+        infix = "_d" if self.settings.build_type == "Debug" else ""
+        self.copy("*.exe", src=build_path, dst=os.path.join(self.package_folder, self._msvc_install_subprefix))
+        self.copy("*.dll", src=build_path, dst=os.path.join(self.package_folder, self._msvc_install_subprefix))
+        self.copy("*.pyd", src=build_path, dst=os.path.join(self.package_folder, self._msvc_install_subprefix, "DLLs"))
+        self.copy("python{}{}.lib".format(self._version_major_minor, infix), src=build_path, dst=os.path.join(self.package_folder, self._msvc_install_subprefix, "libs"))
+        self.copy("*", src=os.path.join(self._source_subfolder, "Include"), dst=os.path.join(self.package_folder, self._msvc_install_subprefix, "include"))
+        self.copy("pyconfig.h", src=os.path.join(self._source_subfolder, "PC"), dst=os.path.join(self.package_folder, self._msvc_install_subprefix, "include"))
+        self.copy("*.py", src=os.path.join(self._source_subfolder, "lib"), dst=os.path.join(self.package_folder, self._msvc_install_subprefix, "Lib"))
+        tools.rmdir(os.path.join(self.package_folder, self._msvc_install_subprefix, "Lib", "test"))
+
+        self.run("{} -c \"import compileall; compileall.compile_dir('{}')\"".format(os.path.join(build_path, self._cpython_interpreter_name), os.path.join(self.package_folder, self._msvc_install_subprefix, "Lib").replace("\\", "/")),
+                 run_environment=True)
+
+    def package(self):
+        self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
+        if self.settings.compiler == "Visual Studio":
+            if self._is_py2:
+                self._msvc_package_copy()
+            else:
+                self._msvc_package_layout()
+        else:
+            autotools = self._configure_autotools()
+            autotools.install()
+            tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+            tools.rmdir(os.path.join(self.package_folder, "share"))
+
+            if self._is_py2:
+                scripts = ("2to3", "idle", "pydoc", "python{}-config".format(self._version_major_minor), "smtpd.py")
+            else:
+                script_prefixes = ("2to3-", "easy_install-", "idle", "pydoc", "pyvenv-", "pip")
+                scripts = tuple(os.path.join(self.package_folder, "bin", sp + self._version_major_minor) for sp in script_prefixes)
```

Comment:
pip3 is not a symlink to pip3.x for some reason, so the #! is not getting updated for it.

Reply:
I think I addressed this problem with the latest commit.

Reply:
Your not globing, so now neither of the pip scripts are getting updated...

Reply:
I would just do
```python
script_prefixes = ("2to3-", "easy_install-", "idle", "pydoc", "pyvenv-", "pip")
scripts = [os.path.join(self.package_folder, "bin", sp + self._version_major_minor) for sp in script_prefixes]
scripts.append(os.path.join(self.package_folder, "bin", "pip3") # Not symlinked to pip3.x for some reason
```

Reply:
I changed it to iterate over all files in the bin directory.
And replace the shebang on only those non-symlinking files that contain a python shebang.